### PR TITLE
Style  inputs

### LINKED ISSUE #31: style the text inputs
We need a single Input (with its label) that can be used on the three pages (views).
AC:
- The input should look like the one in [figma design](https://www.figma.com/file/4qFFHuORhrqvQA1bleIRa4/TCL-44?node-id=0%3A1)
- The input should be reusable.

Notes: 
- by reusable I mean that it should not has hard-coded values. For example, the label's text should be passed as props and not hard-coded.

### PR DESCRIPTION
**Pull request by redapy**
## Description
This PR styles the text inputs on the three pages (views).
- Added the `@apply` directive inside an `input` class.
- Used the `input` class to style all the text inputs.

Notes: 
- I didn't create a new reusable component because our inputs take a ref, and it will be kinda complicated (and unnecessary) to pass refs as props.
- I used blue borders, I think they look good :). 

## Related Issue
closes #31 

## Acceptance Criteria

- [x] The input should look like the one in [figma design](https://www.figma.com/file/4qFFHuORhrqvQA1bleIRa4/TCL-44?node-id=0%3A1)

- [ ] The input should be reusable.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before

before

### After
after

## Testing Steps / QA Criteria
- From your terminal, pull down this branch with `git pull origin rb-create-reusable-styled-input-component` and check that branch out with `git checkout rb-create-reusable-styled-input-component`.
- you can see that the inputs are looking almost the same as the figma design

### CODE REVIEW

**Review by hellodeborahuk — APPROVED** [review]
Looks good to me. We'll need to look at the width of the input and the items once they're done to see how they look all together.

File: src/index.css [hunk 1]
```diff
@@ -5,3 +5,7 @@
 .active {
   font-weight: bold;
 }
+
+.input {
+  @apply border-2 border-blue-300 rounded-full px-4 py-2;
+}
```
Comment (hellodeborahuk) [inline src/index.css:11]: Great use of @apply

**Comment by redapy** [comment]
you are right @hellodeborahuk! I guess it's better to do the width of the inputs, and also other items when we style the whole page. So we can use their parents' width to set their width.

File: src/index.css (hunk 1 above)
Comment (JimeBlue) [reply src/index.css:11]: I agree! what an elegant approach using **@apply**. Your implementation is beautifully clear 👏

**Pull request merged**